.github/PULL_REQUEST_TEMPLATE/new_loader.md

```diff
@@ -0,0 +1,32 @@
+### Title
+ 
+ Please use the following title: "Adding loader for MyDATASET". If your pull request is work in progress, change your title to "[WIP] Adding loader for MyDATASET" to avoid reviews while the loader is not ready.
+
+### Description
+
+Please include the following information in the description of the dataset:
+
+- [ ] Describe annotations included in the dataset
+- [ ] Indicate the size of the datasets (e.g. number files and duration, hours)
+- [ ] Mention the origin of the dataset (e.g. creator, institution)
+- [ ] Describe the genres included in the dataset
+- [ ] Indicate any relevant papers related to the dataset
+- [ ] Include a description on openness/license of the dataset (e.g. is the audio downloadable?) 
```

Comment:
"Include a description about how the data can be accessed and the license it uses (if applicable)."